__tests__/fetcher/api_mockdata.ts

```diff
@@ -0,0 +1,2851 @@
ubtrahieren](/1619)."}],[{"col":24,"content":"###Vorgehensweise mit schriftlicher Subtraktion"}],[{"col":6,"content":"%%6,623-4,71\\\\\\\\%%"},{"col":18,"content":"Schreibe die Zahlen so untereinander, dass die **Kommas direkt untereinander** sind. "}],[{"col":6,"content":"%%\\\\begin{array}{l}\\n\\\\hphantom{ -\\\\; } 6\\\\color{red}{,}623\\\\\\\\\\n-\\\\;4\\\\color{red}{,}71\\\\\\\\\\n\\\\\\\\\\n\\\\end{array}%%"},{"col":18,"content":"Falls die Zahlen unterschiedlich viele Stellen nach dem Komma haben, müssen fehlende Nachkommastellen mit **Nullen aufgefüllt werden**."}],[{"col":6,"content":"%%\\\\begin{array}{l}\\n\\\\hphantom{ -\\\\; } 6,623\\\\\\\\\\n-\\\\;4,71\\\\color{red}{0}\\\\\\\\\\n\\\\\\\\\\n\\\\end{array}%%"},{"col":18,"content":"Als nächstes wird [schriftlich subtrahiert](/1507) ohne die Kommas zu beachten. Im nächsten Schritt werden diese wieder berücksichtigt."}],[{"col":6,"content":"%%\\\\begin{array}{l}\\n\\\\hphantom{ -\\\\; } \\\\overset{5}{\\\\not6},\\\\overset{16}{\\\\not6}\\\\;2\\\\;3\\\\\\\\\\n\\\\underline{-\\\\;\\\\;4,\\\\;7\\\\;1\\\\;0}\\\\\\\\\\n\\\\hphantom{-\\\\;}\\\\;1\\\\,\\\\ \\\\;9\\\\;1\\\\;3\\n\\\\\\\\\\n\\\\end{array}%%"},{"col":18,"content":"Im Ergebnis wird jetzt das Komma **direkt unter die anderen** gesetzt."}],[{"col":6,"content":"%%\\\\begin{array}{l}\\n\\\\hphantom{ -\\\\; } \\\\overset{5}{\\\\not6},\\\\overset{16}{\\\\not6}\\\\;2\\\\;3\\\\\\\\\\n\\\\underline{-\\\\;\\\\;4,\\\\;7\\\\;1\\\\;0}\\\\\\\\\\n\\\\hphantom{-\\\\;}\\\\;1\\\\color{red}{,}\\\\;9\\\\;1\\\\;3\\n\\\\\\\\\\n\\\\end{array}%%"},{"col":18,"content":""}],[{"col":24,"content":"/// Übungsaufgaben\\nBerechne die folgenden Subtraktionen und gib das Ergebnis mit der Tastatur in das blaue Feld ein. \\n\\n>[Beispiel 4](/21095)\\n>[Beispiel 5](/21101)\\n>[Beispiel 6](/21107)\\n\\nWeitere Übungsaufgaben: [Aufgaben zur Addition und Subtraktion von Dezimalbrüchen](/25111)\\n\\n///\\n\\n\\n/// Vertiefung: Warum funktioniert dieses Verfahren?\\nMan sieht leicht wieso dieses Verfahren funktioniert, wenn man die Dezimalbrüche in [Brüche umwandelt](/27800) und das [Kommutativ-](/1679) und [Assoziativgesetz](/1671) nutzt.\\n####Beispiel\\n\\n$$\\\\begin{array}{l}\\\\begin{array}{l}6,623-4,71=\\\\left(6+\\\\frac6{10}+\\\\frac2{100}+\\\\frac3{1000}\\\\right)-\\\\left(4+\\\\frac7{10}+\\\\frac1{100}\\\\right)\\\\\\\\=6-4+\\\\left(\\\\frac6{10}-\\\\frac7{10}\\\\right)+\\\\left(\\\\frac2{100}-\\\\frac1{100}\\\\right)+\\\\frac3{1000}\\\\end{array}\\\\\\\\=2-\\\\frac1{10}+\\\\frac1{100}+\\\\frac3{1000}\\\\\\\\=1,913\\\\\\\\\\\\end{array}$$\\n///"}],[{"col":24,"content":"###Video zur Subtraktion von Dezimalbrüchen\\n\\n>[Subtraktion von Dezimalbrüchen](/51295)"}],[{"col":24,"content":"###Vorgehensweise mithilfe von Umrechnung zu Brüchen\\n\\nStatt direkt mit den Dezimalbrüchen zu rechnen, kann man diese auch [in Brüche umwandeln](/28296)  und anschließend [subtrahieren](/1619) .\\n\\nDieses Verfahren verwendet man vor allem bei [periodischen Brüchen](/2115)."}]]',
```

Comment:
```suggestion
JSON.stringify([[{ col: 24, content: "## Hello World"}]])
```

It is better to have a small example than a large real world example. This is my screen when I am at this line:

![2020-08-30-135803_913x677_scrot](https://user-images.githubusercontent.com/1327215/91658495-e552f780-eac8-11ea-8c69-7f7397b35210.png)

With JSON.stringify and self-made example content it is easier to reason about:

![2020-08-30-140113_532x155_scrot](https://user-images.githubusercontent.com/1327215/91658546-61e5d600-eac9-11ea-91a8-1862220f8cd7.png)